Tests/AdyenTests/Card Tests/CardComponentTests.swift

```diff
@@ -968,6 +968,134 @@ class CardComponentTests: XCTestCase {
         waitForExpectations(timeout: 10, handler: nil)
     }
     
+    func testClear_shouldResetPostalCodeItemToEmptyValue() throws {
+        // Given
+        let method = CardPaymentMethod(type: "bcmc",
+                                       name: "Test name",
+                                       fundingSource: .credit,
+                                       brands: ["visa", "amex", "mc"])
+        var config = CardComponent.Configuration()
+        config.billingAddressMode = .postalCode
+        let sut = CardComponent(paymentMethod: method,
+                                configuration: config,
+                                apiContext: Dummy.context)
+        // When
+        sut.clear()
```

Comment:
there has to be some values in the item.value first before clearing for proper testing, right?